### PR TITLE
Test3

### PR DESCRIPTION
**Pull request by Shantanu-Bakshi**
I have added the the timer as required with
1.  it shows time remaining to the next class
2.  is hidden if the time remaining is less than 24 hours
kindly accept my PR  .

### CODE REVIEW

**Comment by hirenchalodiya1** [comment]
@Shantanu-Bakshi, 
- first pull all commit from `master`
- change the title of PR
- mention which issues you are solving.

**Comment by DarshitJain04** [comment]
> I have added the the timer as required with
> 
> 1. it shows time remaining to the next class
> 2. is hidden if the time remaining is less than 24 hours
>    kindly accept my PR  .

@Shantanu-Bakshi also add necessary screenshots whenever you make UI changes

File: src/pages/Background/index.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title></title>
-  </head>
 
-  <body></body>
-</html>
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
```
Comment (DarshitJain04) [inline src/pages/Background/index.html:5]: Do not make any changes to html/css files
Also adding links can cause Content Security Policy issues

Try looking for how to add fonts in react apps or keep the default font. Upto you

File: src/pages/Popup/components/AlarmUp.jsx [hunk 2]
```diff
@@ -11,13 +11,15 @@ import Zoom from '@material-ui/core/Zoom';
 import PersonAddIcon from '@material-ui/icons/PersonAdd';
 import CloseIcon from '@material-ui/icons/Close';
 import DoneIcon from '@material-ui/icons/Done';
-
+import Timer from "./timer.jsx";
+import './timerstyle2.css';
```
Comment (DarshitJain04) [inline src/pages/Popup/components/AlarmUp.jsx:15]: Don't create new css files for styling purpose
Add styles in the JSX file itself. Here is something you can use for your reference:
https://github.com/devlup-labs/meet-scheduler/blob/9e37ebe3578f29de49acde4297f4e24595fab4e9/src/pages/Popup/components/StudentForm.jsx#L15

File: src/pages/Popup/components/StudentForm.jsx [hunk 3]
```diff
@@ -11,7 +11,8 @@ import ClearIcon from '@material-ui/icons/Clear';
 import * as sheet from '../scripts/sheetapi.js';
 import { AddAlarm_click } from '../scripts/alarm.js';
 import { getAllDataFromStorage } from '../scripts/storage.js';
-
+import Timer from "./timer.jsx";
+import './timerstyle2.css';
```
Comment (DarshitJain04) [inline src/pages/Popup/components/StudentForm.jsx:15]: Same here
Don't use external css files

File: src/pages/Popup/index.html [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
-  </body>
-</html>
+<head>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+  <meta charset="UTF-8" />
+  <title></title>
+</head>
+
+<body class="chrome-extension-react">
+  <div id="extension-app-container"></div>
```
Comment (DarshitJain04) [inline src/pages/Popup/index.html:11]: Don't make any changes to any HTML/CSS files
Make any necessary changes in the JSX files itself
All the changes are bundled and added to the html files

**Review by DarshitJain04 — CHANGES_REQUESTED** [review]

**Comment by DarshitJain04** [comment]
@Shantanu-Bakshi all the changes that you make in the jsx files are bundled and added to the html files. You don't need to make any changes to the html files. 
Also remove the links/ imports from the html file and in case you want to use different fonts, check how to add fonts in react apps
Remove the css files you have created and add the styles in the files itself ( Check my comments, I have added a link for your reference )
In short let the existing html and css files (not the ones you added/changed) remain as it is

Also, use prettier to format your code

**Review by hirenchalodiya1 — CHANGES_REQUESTED** [review]
Pull latest from `devlup-labs:master` to resolve conflict in `package-lock.json`

**Comment by hirenchalodiya1** [comment]
@Shantanu-Bakshi any updates in this PR?

**Comment by Shantanu-Bakshi** [comment]
for resolving issues i've made a new pull request and am therefore closing this one
